CollectionMonitor.cpp

```diff
@@ -38,22 +38,25 @@ void CollectionMonitor::run() {
     Logger::Info("CollectionMonitor started");
 
     if (_netlink.Open(nullptr) != 0) {
-        Logger::Error("AuditRulesMonitor: Could not open NETLINK connect, exiting");
+        Logger::Error("CollectionMonitor: Could not open NETLINK connect, exiting");
         return;
     }
 
     do {
         auto now = std::chrono::steady_clock::now();
 
-        uint32_t audit_pid = -1;
+        if (_pause_collector_check && now - _pause_time > std::chrono::seconds(3600)) {
+            _pause_collector_check = false;
```

Comment:
>_pause_collector_check = false; [](http://example.com/codeflow?start=12&length=31)

Consider to reset _pause_time as well

Reply:
The _pause_time is only evaluated if _pause_collector_check is true, and _pause_time is set at the same time as _pause_collector_check is set to true.